src/main/java/org/zaproxy/zap/extension/hud/tutorial/pages/AlertNotificationsPage.java

```diff
@@ -69,11 +69,11 @@ public void setTaskCompleted(boolean taskCompleted) {
                                                         alertRisk,
                                                         Alert.CONFIDENCE_LOW,
                                                         Constant.messages.getString(
-                                                                "hud.tutorial.page.alertsnotifications.alert.title."
+                                                                "hud.tutorial.page.alertnotifications.alert.title."
```

Comment:
This will break any localizations (I generally ignore localizations when calculating fixes, but I can apply fixes to localizations if desired)

Reply:
Doesnt look like there are any that you havnt changed, so this is good :)